README.md

```diff
@@ -64,8 +63,24 @@ Below are the recognized properties for the Driver section, followed by an examp
   CoapBindAddr = '0.0.0.0'
   # Choose 'PSK' or 'NoSec'
   SecurityMode = 'PSK'
-  # Key is up to 16 arbitrary bytes; must be base64 encoded here
-  PskKey = 'ME42aURHZ3Uva0Y0eG9lZw=='
+```
+
+### Secrets
+
+If configured for PSK mode, a key must be stored in the service's secret store:
+| Secret name | Value                                                                             |
+|-------------|-----------------------------------------------------------------------------------|
+| PskKey      | Pre-shared key. Accepts only a single key, ignored in NoSec mode.                 |
+
+For example if using insecure mode (secrets in configuration file):
+
+```
+  [Writable.InsecureSecrets]
+    [Writable.InsecureSecrets.device-coap]
+      path = "device-coap"

```

Comment:
For consistency the path should be the name of the secret not the service name. Suggest using PSK. 
In secure mode the service should be part of the base path that this `path` is appended to it like:
```
.../device-coap/psk
```
`.../device-coap` is the secret store in Vault for Device Coap.
`.../device-coap/psk` is a Secret within the Device Coap secret store in vault.

The path in the InsucureSecrets needs to mimic that used for Vault.